sympy/concrete/logic/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .boolalg import (to_cnf, to_dnf, to_nnf, And, Or, Not, Xor, Nand, Nor, Implies,
```

Comment:
I think something went wrong, I tried changing only matrices and the test file. @jksuom can you have a look on those, I'll remove this commit later.